cluster-up/cluster/kind/common.sh

```diff
@@ -141,6 +141,35 @@ function _fix_node_labels() {
     done
 }
 
+function _get_cri_bridge_mtu() {
+  docker network inspect -f '{{index .Options "com.docker.network.driver.mtu"}}' bridge
+}
+
+function _patch_calico_manifest() {
+  local -r calico_manifest="$1"
+  local -r calico_diff="$2"
+  local -r destination_dir="$3"
```

Comment:
We don't need a generic function here, just put the details here.

Reply:
I removed them